packages/react/src/experimental/Widgets/Content/ClockIn/ClockInControls/index.tsx

```diff
@@ -106,10 +106,13 @@ export function ClockInControls({
   const [breakTypePickerOpen, setBreakTypePickerOpen] = useState(false)
 
   const handleClickBreakButton = () => {
-    if (breakTypeOptions?.length && !breakTypePickerOpen) {
-      setBreakTypePickerOpen(true)
-    } else if (!breakTypeOptions?.length) {
-      onBreak?.()
+    if ((breakTypeOptions?.length ?? 0) > 1) {
```

Comment:
is there a way to always have breaktypeOptions set? do you use the empty state somewhere?